data-reporting-service/src/main/resources/db/rdb/023-sp_nrt_investigation_postprocessing-001.sql

```diff
@@ -1,501 +1,643 @@
-        inv.IMPORT_FRM_STATE,
-        inv.IMPORT_FRM_CNTY,
-        inv.IMPORT_FRM_CITY,
-        inv.EARLIEST_RPT_TO_CDC_DT,
-        inv.RPT_SRC_CD,
-        inv.IMPORT_FRM_CNTRY_CD,
-        inv.IMPORT_FRM_STATE_CD,
-        inv.IMPORT_FRM_CNTY_CD,
-        inv.IMPORT_FRM_CITY_CD,
-        inv.DIAGNOSIS_DT,
-        inv.HSPTL_ADMISSION_DT,
-        inv.HSPTL_DISCHARGE_DT,
-        inv.HSPTL_DURATION_DAYS,
-        inv.OUTBREAK_IND,
-        inv.HSPTLIZD_IND,
-        inv.INV_STATE_CASE_ID,
-        inv.CITY_COUNTY_CASE_NBR,
-        inv.TRANSMISSION_MODE,
-        inv.RECORD_STATUS_CD,
-        inv.PATIENT_PREGNANT_IND,
-        inv.DIE_FRM_THIS_ILLNESS_IND,
-        inv.DAYCARE_ASSOCIATION_IND,
-        inv.FOOD_HANDLR_IND,
-        inv.INVESTIGATION_DEATH_DATE,
-        inv.PATIENT_AGE_AT_ONSET,
-        inv.PATIENT_AGE_AT_ONSET_UNIT,
-        inv.INV_ASSIGNED_DT,
-        inv.DETECTION_METHOD_DESC_TXT,
-        inv.ILLNESS_DURATION,
-        inv.ILLNESS_DURATION_UNIT,
-        inv.CONTACT_INV_COMMENTS,
-        inv.CONTACT_INV_PRIORITY,
-        inv.CONTACT_INFECTIOUS_FROM_DATE,
-        inv.CONTACT_INFECTIOUS_TO_DATE,
-        inv.CONTACT_INV_STATUS,
-        inv.PROGRAM_AREA_DESCRIPTION,
-        inv.ADD_TIME,
-        inv.LAST_CHG_TIME,
-        inv.INVESTIGATION_ADDED_BY,
-        inv.INVESTIGATION_LAST_UPDATED_BY,
-        inv.REFERRAL_BASIS,
-        inv.CURR_PROCESS_STATE,
-        inv.INV_PRIORITY_CD,
-        inv.COINFECTION_ID,
-        inv.OUTBREAK_NAME_DESC
-    FROM #temp_inv_table inv
-             join dbo.nrt_investigation_key k on inv.case_uid = k.case_uid
-    where inv.investigation_key is null;
-
-
-    /* Logging */
-    set @rowcount=@@rowcount
-    if @rowcount > 0
-        update dbo.nrt_batch_log
-        set log_detail=log_detail+' Num of Investigation keys inserted: ' + cast(@rowcount as nvarchar(max))+ '.'
-        where batch_id=@log_id;
-    else
-        update dbo.nrt_batch_log
-        set log_detail=log_detail+' No Investigation Inserts.'
-        where batch_id=@log_id;
-
-
-
-    /* Temp Confimation Method Table*/
-    select distinct
-        nrt.PUBLIC_HEALTH_CASE_UID,
-        i.INVESTIGATION_KEY,
-        nrt.CONFIRMATION_METHOD_CD,
-        nrt.CONFIRMATION_METHOD_DESC_TXT,
-        nrt.CONFIRMATION_METHOD_TIME as CONFIRMATION_DT,
-        cm.CONFIRMATION_METHOD_KEY
-    into #temp_cm_table
-    from dbo.nrt_investigation_confirmation nrt
-             left join dbo.confirmation_method cm on cm.confirmation_method_cd=nrt.confirmation_method_cd
-             left join dbo.investigation i on i.case_uid = nrt.public_health_case_uid
-    where nrt.public_health_case_uid in (select value FROM STRING_SPLIT(@id_list, ','));
-
-
-    -- if confirmation_method_key for the cd exists get the key or insert a new row to rdb.confirmation_method
-
-    /*Update Operation for confirmation_method and group*/
-    update cm
-    set cm.CONFIRMATION_METHOD_DESC = cmt.CONFIRMATION_METHOD_DESC_TXT
+	/* Logging */
```

Comment:
is this due to the formatting change

Reply:
Yes :/ Let me remove the [] and push a new change.